webgui/scripts/ga.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
+    $.ajax({
+        url: 'ga.txt',
+        dataType: 'text',
+        success: function (gaId) {
+            console.log('Google Analytics enabled: ' + gaId);
+            $.getScript('https://www.googletagmanager.com/gtag/js?id=' + gaId);
+
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
```

Comment:
This function is weirdly formatted.

Reply:
Because I have copied the template code:
https://developers.google.com/analytics/devguides/collection/gtagjs

Of course we format it according to our coding conventions 😄

Reply:
Fixed.